source/wp-content/themes/wporg-news-2021/functions.php

```diff
@@ -141,86 +139,6 @@ function modify_archive_title_prefix( $prefix ) {
 	return $prefix;
 }
 
-/**
- * Load the `index.html` template for `w.org/news/page/{n}` requests, rather than `front-page.html`.
- *
- * The design calls for a mix of `front-page.html` and `home.html`/`index.html` functionality. "home" is Core's
- * legacy terminology for the posts index from the pre-CMS days. The design calls it the "all posts" screen.
- *
- * Setting it up in separate files requires this hack, but feels more straight-forward overall. This lets us have
- * static content and the latest posts on the front page, while also preserving clean URLs. Using
- * `show_on_front = page` would change the URLs to `w.org/news/posts/page/2` rather than `w.org/news/page/2`.
```

Comment:
It's true that now the paged URLs are `/news/all-posts/page/2/`, instead of just `/news/page/2/`. Is this still considered a problem?

Reply:
I think it's fine. Having a slug URL structure instead of a query string is the part I was most concerned about.